docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,442 @@
+			"specimenTissueSource": "Blood derived"
+		},
+		"donor": {
+			"submitterDonorId": "DASH-1",
+			"gender": "Male"
+		}
+	}],
+	"files": [{
+		"dataType": "Submitted Reads",
+		"fileName": "dash-1-normal.bam",
+		"fileSize": 24,
+		"fileType": "BAM",
+		"fileMd5sum": "4d6e35e3290ccce08c8c7517dc29d624",
+		"fileAccess": "controlled"
+	}],
+	"read_group_count": 1,
+	"read_groups": [{
+		"file_r1": "dash-1-normal.bam",
+		"file_r2": null,
+		"insert_size": null,
+		"is_paired_end": false,
+		"library_name": "Library-Testing",
+		"platform_unit": "Library-Testing",
+		"read_length_r1": null,
+		"read_length_r2": null,
+		"sample_barcode": null,
+		"submitter_read_group_id": "dash-1-rg-1"
+	}],
+	"experiment": {
+		"submitter_sequencing_experiment_id": "DASH-SE-1",
+		"experimental_strategy": "WGS",
+		"sequencing_center": "",
+		"platform": "ILLUMINA",
+		"platform_model": null,
+		"sequencing_date": null
+	}
+}
+```
+
+</TabItem>
+<TabItem value="tumour">
+
+```json
+{
+	"studyId": "DASH-CA",
+	"analysisType": {
+		"name": "sequencing_experiment"
+	},
+	"samples": [{
+		"submitterSampleId": "dashsa-1.1",
+		"matchedNormalSubmitterSampleId": "dashsa-1",
+		"sampleType": "Amplified DNA",
+		"specimen": {
+			"submitterSpecimenId": "dashsp-1.1",
+			"specimenType": "Primary tumour",
+			"tumourNormalDesignation": "Tumour",
+			"specimenTissueSource": "Cerebellum"
+		},
+		"donor": {
+			"submitterDonorId": "DASH-1",
+			"gender": "Male"
+		}
+	}],
+	"files": [{
+		"dataType": "Submitted Reads",
+		"fileName": "dash-1-tumour.bam",
+		"fileSize": 24,
+		"fileType": "BAM",
+		"fileMd5sum": "d5fd5482b53ee89e71eda8d69cba4bd9",
+		"fileAccess": "controlled"
+	}],
+	"read_group_count": 1,
+	"read_groups": [{
+		"file_r1": "dash-1-tumour.bam",
+		"file_r2": null,
+		"insert_size": null,
+		"is_paired_end": false,
+		"library_name": "Library-Testing",
+		"platform_unit": "Library-Testing",
+		"read_length_r1": null,
+		"read_length_r2": null,
+		"sample_barcode": null,
+		"submitter_read_group_id": "dash-1-rg-2"
+	}],
+	"experiment": {
+		"submitter_sequencing_experiment_id": "DASH-SE-1.2",
+		"experimental_strategy": "WGS",
+		"sequencing_center": "",
+		"platform": "ILLUMINA",
+		"platform_model": null,
+		"sequencing_date": null
+	}
+}
+```
+
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+See github for the same example of a correctly formatted payload from a [normal sample](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/example_payloads/dash1_normal.json) and a [tumour sample](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/example_payloads/dash1_tumour.json).
+
+Depending on the type of sequencing that was done, the `read_groups` section of the payload will need to be adjusted. Examples of correctly formatted `read groups` that meet [the rules](/docs/submission/submitting-molecular-data#file-and-data-validation-rules) for paired-end vs single-end sequencing are are shown here as examples. 
```

Comment:
suggestion
`The following are examples of correctly formatted `read groups` that meet the rules for paired-end vs single-end sequencing:`